thirdeye/thirdeye-frontend/app/pods/manage/alerts/edit/controller.js

```diff
@@ -355,15 +355,19 @@ export default Ember.Controller.extend({
 
   /**
    * Display success banners while model reloads
-   * @method resetView
+   * @method confirmEditSuccess
    * @return {undefined}
    */
-  resetView() {
+  confirmEditSuccess() {
     const that = this;
-    this.send('refreshModel');
+    this.setProperties({
+      isEditAlertSuccess: true,
+      isProcessingForm: false
+    });
     Ember.run.later((function() {
       that.clearAll();
-    }), 2000);
+      that.send('refreshModel');
+    }), 3000);
```

Comment:
any reason why we needed to bump this up?

Reply:
No, doesn't make much difference.